src/components/AgreementSections/ControllingLicense.js

```diff
@@ -83,6 +91,18 @@ export default class ControllingLicense extends React.Component {
               id="controlling-license-current-amendments"
               license={licenseRecord}
               renderStatuses
+              renderWarnings
+              renderNotes
+            />
+          </KeyValue>
+          : null
+        }
+        { unsetAmendments.length ?
```

Comment:
Why can't we just `this.renderAmendments` here? It seems like all we're doing is moving it into the card. Can tweak that method if it doesn't do exactly what you need.

Reply:
We can do that here, I was simply following the example above, where currentAmendments are rendered directly. Of course I can also make that part use `this.renderAmendments`, which if I'm already tweaking that method I think I will do, as little repeated code as possible.

Reply:
Hmmm, yeah good catch. I'm honestly not sure why I didn't use `renderAmendments` for the current amendments and instead wrote it out. Nothing stands out. You're right there's a tiny performance hit but the benefit in terms of making the code easier to maintain is big enough that it's worth it IMO.